tests/models/bert/test_modeling_bert.py

```diff
@@ -622,6 +629,79 @@ def test_torchscript_device_change(self):
                 loaded = torch.jit.load(os.path.join(tmp, "bert.pt"), map_location=torch_device)
                 loaded(inputs_dict["input_ids"].to(torch_device), inputs_dict["attention_mask"].to(torch_device))
 
+    # This test was copied from the common test_eager_matches_sdpa_generate(), but without low_cpu_mem_usage=True.
```

Comment:
The PRs involving the blocking #29024 was rolled back in main/HEAD for the release. 

I think that task will take longer to complete, since the plan is to add support for low_cpu_mem_usage=True for all those models (~30 models). 

In the meantime, I've overridden the common test in the Bert test, and commented out the low_cpu_mem_usage. It should pass now. I'll remove this copy once I finish adding support for low_cpu_mem_usage in Bert.

Reply:
Yep, it's such a small model that `low_cpu_mem_usage` is not super high priority!